package.json

```diff
@@ -95,6 +94,7 @@
     "mongodb": "^4.10.0",
     "mongodb-uri": "^0.9.7",
     "node-cron": "3.0.0",
+    "node-fetch": "^2.6.12",
```

Comment:
I think the idea was to replace Axios with built-in fetch, not `node-fetch` library. You can see the example of using built-in fetch [here](https://github.com/hyperjumptech/monika/blob/main/packages/notification/channel/pagerduty.ts#L77).